base/src/com/thoughtworks/go/util/SystemEnvironment.java

```diff
@@ -447,6 +447,10 @@ public int getNumberOfConfigMaterialCheckListener() {
         return Integer.parseInt(getPropertyImpl("material.config.check.threads", "2"));
     }
 
+    public int getNumberOfDependencyMaterialCheckListener() {
```

Comment:
Done. No particular reason for not using the ```GoIntSystemProperty```, just went with what was done for other CheckListeners. Changed it to use  ```GoIntSystemProperty```.